internal/proxy/proxy.go

```diff
@@ -129,26 +129,38 @@ func (proxy *Proxy) Close() error {
 func (proxy *Proxy) Start() {
 	proxy.log.Info().Str("name", proxy.Config.Hostname).Msg("starting proxy")
 	var err error
-	// Create the listener
-	//
-	proxy.listener, err = proxy.proxyProvider.GetTLSListener("tcp", ":443")
-	if err != nil {
-		proxy.log.Error().Err(err).Msg("Error Listening on TLS")
-		proxy.Close()
-		return
-	}
 
-	if err = proxy.proxyProvider.Start(); err != nil {
-		proxy.log.Error().Err(err).Msg("Error starting proxy")
-		proxy.Close()
-		return
-	}
+		scheme := proxy.Config.ProxyURL.Scheme
```

Comment:
this will be always https in Docker provider.
We need to add something in docker provider that defines the proxy scheme.
the label scheme that exists already is to target scheme and not to proxy.

Reply:
I understand, if you point me to a proper variable name in the .yaml config i can make a PR for it. Maybe a top-level `ProxyScheme`?